### PR TITLE
Add: Notifiications, Settings, Help page and toastify

### PR DESCRIPTION
**Pull request by saurabhchirde**
Features

1. After new achievements, notifications will be added to the notifications page.
2. User can Change his name, password
3. User can delete his account

help-notification-settings

### CODE REVIEW

File: src/Pages/Help/HelpPage.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              <ol className="list-basic">
+                <li>Avengers</li>
+                <li>Sports</li>
+                <li>Movies</li>
+                <li>Travel</li>
+                <li>Science</li>
+                <li>Technology</li>
+                <li>Animals</li>
+                <li>Language</li>
+              </ol>
```
Comment (PuneetP16) [inline src/Pages/Help/HelpPage.jsx]: Suggestion: you could map the list items here.

**Review by PuneetP16 — APPROVED** [review]
Appreciation: your UI looks good and the accordion too in the help section. Good work, looks good to merge.

**Review by dev-ashishsethi — APPROVED** [review]
looks good

File: src/Pages/Help/HelpPage.jsx (hunk 1 above)
Comment (saurabhchirde) [reply src/Pages/Help/HelpPage.jsx]: Noted

File: src/Pages/Notifications/NotificationPage.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    padding-left: 2px;
+  }
+
+  .all-notifications {
+    margin-top: 2.5rem;
+  }
+
+  .list-stacked img {
+    width: 2.5rem;
+    /* margin-left: 5px; */
```
Comment (deekshithmd) [inline src/Pages/Notifications/NotificationPage.css]: You can remove commented codes

File: src/Pages/Help/HelpPage.jsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                section.
+                <img src={badges} alt="badges" />
+              </p>
+            </div>
+          </details>
+        </div>
+      </div>
+    </div>
+  );
+};
```
Comment (deekshithmd) [inline src/Pages/Help/HelpPage.jsx:95]: UI looks good, It is good to see a lot of P1 and P2 features added

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**